test/cli/kit/system_test.rb

```diff
@@ -32,6 +34,57 @@ def test_split_partial_characters_gives_up_without_error_for_bad_utf8
 
         assert_equal [str, ''], System.split_partial_characters(str)
       end
+
+      def test_system_finds_system_ruby_instead_of_local_script
```

Comment:
I'm currently playing with these tests. There seems to be some flakiness in determining the current working directory, so it may be best to just drop them.

Reply:
Fixed. State leakage from `Dir.chdir`.

Reply:
This test seems less relevant now.

Reply:
I think it's worth keeping. Besides for being a smoke test (i.e., is it even working?), it'll also ensure we don't accidentally resolve the path to a script in the working directory. WDYT?